feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go

```diff
@@ -1171,38 +1171,64 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	d := &oc.Root{}
 	q := d.GetOrCreateQos()
 
+	type qosVals struct {
+		be0, be1, af1, af2, af3, af4, nc1 string
+	}
+
+	qos := qosVals{
+		be0: "BE0",
+		be1: "BE1",
+		af1: "AF1",
+		af2: "AF2",
+		af3: "AF3",
+		af4: "AF4",
+		nc1: "NC1",
+	}
+
+	if dut.Vendor() == ondatra.JUNIPER {
```

Comment:
Where is the deviation that tracks the need for this change?

Reply:
As stated elsewhere, I received confirmation that we are punting on ironing out this non-uniformity. I am tracking it elsewhere

Reply:
same reason as https://github.com/openconfig/featureprofiles/pull/1378, this is not a deviation